tests/fe/fe_rannacher_turek_01.cc

```diff
@@ -121,11 +121,11 @@ void test_interpolation()
     {
       fev.reinit(cell);
 
-      std::vector<double> values(quadrature.size());
+      std::vector<Vector<double> > values(quadrature.size(), Vector<double>(1));
```

Comment:
I had that same thought when looking over the patch. But I needed the space still when I wrote this part ;-)